src/data_structures/stack.spec.js

```diff
@@ -0,0 +1,67 @@
+import { Stack } from "./stack.js";
+
+describe('Stack', () => {
+    it("should return top element on the stack: 'a'", () => {
+        // assign
+        const stack = new Stack();
+
+        // act
+        stack.push('a');
+
+        // assert
+        expect(stack.peek()).toEqual('a');
+    });
+
+    it("should return top element on the stack: true", () => {
+        // assign
+        const stack = new Stack();
+
+        // act
+        stack.push('a');
+        stack.push(1);
+        stack.push(true);
+
+        // assert
+        expect(stack.peek()).toEqual(true);
```

Comment:
what if act would look like this
stack.push(1)
I'm not sure how jasmine works here but IMO toBe should be used

Reply:
basically use toBe whenever possible

Reply:
modified as suggested, personally I don't grasp the difference in specifications of toBe and toEqual:
toEqual(expected)
expect the actual value to be equal to the expected, **using deep equality comparison**.
toBe(expected)
expect the actual value to be === to the expected value.